### DISCUSSION TITLE
auto_archive_duration for thread

### DISCUSSION THREAD

**Discussion by BOXERRMD** [Pycord-Development/pycord#2380] [Q&A] [answered]
Hello,
I seem to have discovered a bug concerning the archiving time of Threads that can be defined at their creation.

`discord.errors.ApplicationCommandInvokeError: Application Command raised an exception: HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In auto_archive_duration: Value "30" is not a valid enum value.`

My code : 

`await ctx.channel.create_thread(name=nom, message=message, auto_archive_duration=30, reason=f'Créer par {ctx.author.name}')
`

I did enter a number as indicated in the documentation, but it doesn't seem to work... Can you help me?

**Comment by Dorukyum** [comment, marked as answer]
You have to choose between 60 (one hour), 1440 (one day), 4320 (3 days) and 10080 (one week). We _should_ add this to the documentation though.

**Reply by BOXERRMD** [reply]
thanks ^^
